demo/run.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 snakemake --keep-going --latency-wait 120 --jobs 100 --cluster-sync "srun -J {rule} --cpus-per-task {threads} --mem {resources.mem_mb}" \
---use-conda --conda-prefix /projects/clc/usr/anaconda/workflow_prototype/lcr-modules-envs/ all
+--use-conda --conda-prefix /projects/clc/usr/anaconda/workflow_prototype/lcr-modules-envs/ all \
```

Comment:
This script should be general. Right now, it's tailored for your usage. Feel free to maintain a copy of a script for your purposes (e.g., `demo/run-clc.sh`) that you omit from the repo. I already updated the script in `master` to achieve this. You should be able to import that version by running this:

```
❯ cat demo/run.sh
#!/bin/bash
snakemake --keep-going --latency-wait 120 --jobs 100 --cluster-sync "srun -J {rule} --cpus-per-task {threads} --mem {resources.mem_mb}" \
--use-conda --conda-prefix /projects/clc/usr/anaconda/workflow_prototype/lcr-modules-envs/ all \
--rerun-incomplete

❯ git checkout master -- demo/run.sh

❯ cat demo/run.sh
#!/bin/bash

# Number of cores set by first command-line argument (default is 4)
NUM_CORES=${1:-4}

snakemake --cores "${NUM_CORES}" --printshellcmds --use-conda all
```